js_modules/dagster-ui/packages/ui-core/src/launchpad/TagEditor.tsx

```diff
@@ -52,6 +53,50 @@ export const TagEditor = ({
     }
   }, [tagsFromSession, open]);
 
+  const {toError, toSave} = validateTagEditState(editState);
```

Comment:
The changes in this file don't affect the TagEditor's behavior, I just split the content of the dialog out into a component that could be re-used to edit "lists of tags" generally